source/lambda/kendraIndexCreator/lambda_function.py

```diff
@@ -0,0 +1,51 @@
+import boto3
```

Comment:
Should we rename the file to CustomResourceKendraIndex?
Since it does take care of creation as well as other events?